packages/atomic/stencil.config.ts

```diff
@@ -180,63 +254,38 @@ export const config: Config = {
       ],
     }),
     replace(),
+    isCDN &&
+      replaceWithASTPlugin({
+        replacements: generateReplaceValues(),
+      }),
```

Comment:
Why is the rollup plugin in the `plugins` field for stencil plugins ? Shouldn't it be in `rollupPlugins` ?

Reply:
You would think that, I did too at first. It really has to be there to work. I'm not 100% sure why though...

Note that the replace() plugin just before is also a rollup plugin.